### DISCUSSION TITLE
论文模板由 v2.2 更新至 v2.3 方法

### DISCUSSION THREAD

**Discussion by cyzhou1221** [nuaatug/nuaathesis#59] [Q&A] [answered]
@yzwduck, 请问是替换根目录下的 `nuaathesis.cls` 文件还是 `demo_chs` 下的 `nuaathesis.cls` 文件？

**Comment by yzwduck** [comment, marked as answer]
发布的压缩包里，所有 `nuaathesis.cls` 的内容是相同的，可以用任何一个替换。

**Reply by cyzhou1221** [reply]
okay!
